nakadi-java-avro/src/test/java/nakadi/avro/AvroPublishingBatchSerializerTest.java

```diff
@@ -0,0 +1,158 @@
+package nakadi.avro;
+
+import com.fasterxml.jackson.dataformat.avro.AvroMapper;
+import com.fasterxml.jackson.dataformat.avro.AvroSchema;
+import nakadi.BusinessEventMapped;
+import nakadi.EventMetadata;
+import nakadi.SerializationContext;
+import org.apache.avro.Schema;
+import org.junit.Assert;
+import org.junit.Test;
+import org.zalando.nakadi.generated.avro.PublishingBatch;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Collections;
+import java.util.Objects;
+
+public class AvroPublishingBatchSerializerTest {
+
+    private final String schema = "{\"type\":\"record\",\"name\":\"BusinessPayload\",\"fields\":[{\"name\":\"a\",\"type\":[\"null\",\"string\"]},{\"name\":\"b\",\"type\":[\"null\",\"string\"]},{\"name\":\"id\",\"type\":[\"null\",\"string\"]}]}";
+
+    @Test
+    public void testToBytes() throws IOException {
+        BusinessPayload bp = new BusinessPayload("22", "A", "B");
+        BusinessEventMapped<BusinessPayload> event =
+                new BusinessEventMapped<BusinessPayload>()
+                        .metadata(EventMetadata.newPreparedEventMetadata())
+                        .data(bp);
+
+        AvroPublishingBatchSerializer avroPublishingBatchSerializer = new AvroPublishingBatchSerializer(new AvroMapper());
+        byte[] bytesBatch = avroPublishingBatchSerializer.toBytes(
+                new TestSerializationContext("ad-2022-12-13", schema, "1.0.0"),
+                Collections.singletonList(event)
+        );
+
+        PublishingBatch publishingBatch = PublishingBatch.fromByteBuffer(ByteBuffer.wrap(bytesBatch));
+        byte[] eventBytes = publishingBatch.getEvents().get(0).getPayload().array();
+        BusinessPayload actual = new AvroMapper().reader(
+                new AvroSchema(new Schema.Parser().parse(schema)))
+                .readValue(eventBytes, BusinessPayload.class);
+
+        Assert.assertEquals(bp, actual);
+    }
+
+    @Test
+    public void testToBytes() throws IOException {
```

Comment:
`testToBytes` method is duplicated [here](https://github.com/dehora/nakadi-java/pull/383/files#diff-df9db23c6f29dfa3d1db0d33af9e19f64beebb99d744e8e7edfe2ce4a6e23941R23)